### DISCUSSION TITLE
Guide?

### DISCUSSION THREAD

**Discussion by EpocKis** [P4TOLINO06/FSR3.0-Mod-Setup-Utility#7] [Q&A] [answered]
Hi! where is the Guide? i cant find it/them

**Comment by P4TOLINO06** [comment, marked as answer]
Hi, just check the FSR Guide box inside the utility, or click on "Guide"
if you're using the enhanced version

Em seg., 11 de nov. de 2024 às 11:18, EpocKis ***@***.***>
escreveu:
